src/lib/api.js

```diff
@@ -0,0 +1,40 @@
+const base = 'http://localhost:5001';
```

Comment:
Hard-coded?

Reply:
At the moment yes,
Its in my todo list and will push those changes shortly thanks!